feature-libs/organization/administration/components/unit/form/unit-form.component.html

```diff
@@ -48,26 +48,32 @@
         [control]="form.get('approvalProcess.code')"
       ></cx-form-errors>
     </label>
-
-    <label *ngIf="form.get('parentOrgUnit')" formGroupName="parentOrgUnit">
-      <span class="label-content required">{{
-        'orgUnit.form.parentOrgUnit' | cxTranslate
+    <ng-container *ngIf="form.get('parentOrgUnit'); else parentOrgUnitNotes">
+      <label *ngIf="form.get('parentOrgUnit')" formGroupName="parentOrgUnit">
```

Comment:
It's unnecessary to check conditions inside the block if it was already checked upstream.
```suggestion
      <label formGroupName="parentOrgUnit">
```

Reply:
oh.. i forgot to remove inside condtion... thanks